covid19/templates/admin/covid19_admins_page.html

```diff
@@ -17,5 +19,22 @@
 {% endif %}
 
 {% block content %}
-<p>Template admin</p>
+<h1>Gerenciamento de dataset Covid19</h1>
+
+<div id="content-main">
+  <p>Olá {{request.user.username}}! Nesta página você poderá desempenhar funções administrativas em cima dos dados da Covid19.</p>
+
+  <div class="module aligned">
+    <div class="form-row">
+      <div>
+        <label><b>Atualizar totais dos estados:</b></label>
+        <div class="object-tools">
+          <a href="{% url 'admin:covid19_management' %}?action=update_state_totals">Aualizar Total dos Estados</a>
+        </div>
+        <div class="help">Ao clicar no botão acima, o sistema irá criar novas planilhas para os estados somente com a entrada dos dados totais. Essas planilhas extrairão os dados de acordo com as entradas da planilha <a href="https://docs.google.com/spreadsheets/d/17mmfgPAcVCeHW3548BlFtuurAvF3jeffRVO1NW7rVgQ/edit#gid=0" target="_blank">brio_totais</a>. Somente as entradas que tiverem a coluna <b>status</b> com o valor <b>ok</b> serão consideradas durante a atualização.</div>
```

Comment:
Além da sugestão abaixo, teria que adicionar no contexto a variável `state_totals_url` - o ideal seria centralizá-la em algum lugar (não precisa ser no settings, mas se for, é melhor).

```suggestion
        <div class="help">
          Ao clicar no botão acima, o sistema irá criar novas planilhas para os estados
          somente com a entrada dos dados totais.  Essas planilhas extrairão os dados
          de acordo com as entradas da planilha
          <a href="{{ state_totals_url }}" target="_blank">brio_totais</a>. Somente as
          entradas que tiverem a coluna <b>status</b> com o valor <b>ok</b> serão
          consideradas durante a atualização.
        </div>
```